pyiceberg/avro/reader.py

```diff
@@ -201,6 +201,15 @@ def skip(self, decoder: BinaryDecoder) -> None:
         decoder.skip(16)
 
 
+class UnknownReader(Reader):
+    def read(self, decoder: BinaryDecoder) -> None:
+        _ = decoder.read_bytes()
+        return None
+
+    def skip(self, decoder: BinaryDecoder) -> None:
+        decoder.skip_bytes()
```

Comment:
I missed this one, this should be a no-op as well:
```suggestion
    def read(self, decoder: BinaryDecoder) -> None:
        return None

    def skip(self, decoder: BinaryDecoder) -> None:
        pass
```